NOnion/Network/TorCircuit.fs

```diff
@@ -25,8 +25,8 @@ type CircuitNodeDetail =
     | FastCreate
     | Create of
         EndPoint: IPEndPoint *
-        NTorOnionKey: array<byte> *
-        IdentityKey: array<byte>
+        NTorOnionKey: NTorOnionKey *
+        IdentityKey: IdentityKey
```

Comment:
Maybe we should change IdentityKey to Fingerprint everywhere 🤔 since technically this is the digest of identity key.

Reply:
Changed IdentityKey to Fingerprint and added a comment. Changed some related identifiers.